lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -228,6 +233,31 @@ def _on_relation_changed(self, event) -> None:
 
         self.charm.unit.status = ActiveStatus()
 
+    def _on_relation_broken(self, event) -> None:
+        # Only relation_deparated events can check if scaling down
+        departed_relation_id = event.relation.id
```

Comment:
this variable seems to be only used once on the subsequent condition - safe to drop and refer to `event.relation.id`